src/pages/Selector/container.jsx

```diff
@@ -7,13 +7,14 @@ const SelectorContainer = Wrapped =>
     state = { employees: [], templates: [] };
 
     componentDidMount() {
+      /* eslint-disable no-shadow */
```

Comment:
Can we do this in an eslint rc file or something? Looks just a bit messy in my opinion